### ISSUE TITLE
Insurance

### ISSUE THREAD

**Issue by ghost** [dwyl/hq#256]
[QDOS](https://www.qdoscontractor.com/) has expired

[Digital Risks](https://digitalrisks.co.uk/cover-for/software-and-app-development/) Quote:

Employers Liability £10,000,000
Public and Products Liability £2,000,000

Total £10.56/month

They also offer insurance for:
Office contents and portable equipment
Professional indemnity
Cyber security
Commercial legal protection

Links to #211

**Comment by markwilliamfirth** [comment]
@iteles @nelsonic Can I have confirmation that none of the below have occurred to you?

in the last ten years been declared bankrupt or insolvent or been the subject of bankruptcy proceedings or insolvency proceedings or been disqualified from being a company director
in the last ten years been the subject of a County Court Judgement, an Individual Voluntary Arrangement, a Company Voluntary Arrangement or a Sheriff Court Decree
had a business insurance proposal declined, renewal refused, insurance cancelled or special terms applied
been convicted of, or charged with (but not yet tried for), or given an official police caution in respect of any criminal offence (other than a motoring offence) which is not spent under the Rehabilitation of Offenders Act
never been subject to an investigation by HM Revenue and Customs which has resulted in a prosecution

**Comment by markwilliamfirth** [comment]
And please confirm that you have never been:

- prosecuted by the Health and Safety Executive or been notified that a prosecution will be made and/or received Prohibition or Improvement Notices for breaches of health and safety
- prosecuted by an enforcing authority or been notified that a prosecution will be made and/or received Enforcement or Prohibition Notices for breaches in fire safety
- prosecuted by the Crown Prosecution Service under the Corporate Manslaughter and Corporate Homicide Act 2007

**Comment by markwilliamfirth** [comment]
And:

- forced to issue a product recall by the enforcing authorities
- prosecuted under Product Safety legislation
- prosecuted by the Environment Agency and/or been subject to Civil Sanctions and/or been required to pay clean-up costs following a pollution incident

**Comment by markwilliamfirth** [comment]
And that you haven't:

- You haven't made any claims, or had any incidents which could have resulted in a claim, in the past 3 years

**Comment by iteles** [comment]
@markwilliamfirth All the above confirmed for us both.

**Comment by markwilliamfirth** [comment]
We now have the following insurance with Digital Risks

Employers Liability £10,000,000
Public and Products Liability £2,000,000

Total £10.56/month

This was backdated to a start date of 30 January 2017.

**Comment by nelsonic** [comment]
@markwilliamfirth thanks for sorting this. ❤️  ✅

**Comment by markwilliamfirth** [comment]
Sorry should have commented(!)

Decided to leave it open and move to priority 2 because we still need to discuss whether we want:

Office contents and portable equipment
Professional indemnity
Cyber security
Commercial legal protection

Will revisit this after priority 1s :)

**Comment by rub1e** [comment]
I've asked our insurer, Digital Risks, and a couple of other ones which specialise in co-working spaces to let me know what info they need to provide a quote (presume we'll need to start with https://github.com/dwyl/hq/issues/81#issuecomment-283029702)

**Comment by rub1e** [comment]
**Professional indemnity renewal** - Digital Risks have offered us the following levels for renewal:
* £500,000 limit of indemnity cover for £26.60 including tax (monthly premium)
* £1m LOI for £36.75 incl. tax 
* £2m LOI for £62.48 incl. tax

This is around half of the cheapest quote I got off GoCompare, so it's looking good

**Contents** - I need to do an inventory, and answer a few other questions before we can proceed.

Call scheduled for 14/02/18

**Comment by iteles** [comment]
@rub1e Let's go with the £2m renewal for professional indemnity 👍 
Thanks for picking this up!

**Comment by rub1e** [comment]
Apologies for not updating this sooner - we have professional indemnity insurance and the documents are saved in `Legals (private) > Insurance`

Keeping this open until Contents is settled

**Comment by rub1e** [comment]
We'll need contents insurance when we move to Space4 and the process will be made easier when we go through FocusHub and inventory everything (https://github.com/dwyl/focus-hub/issues/136).

I'll recontact Digital Risks when we do this

**Comment by rub1e** [comment]
Digital Risks have kindly given us free `Contents Insurance` (£100 excess, which was the minimum), to compensate us for having ignored several of my emails for several weeks

The policy documents are saved `Legals (private) > Insurance`

Closing now - will re-open if any documents need updating

**Comment by iteles** [comment]
DD cancelled as no longer needed from this company.
